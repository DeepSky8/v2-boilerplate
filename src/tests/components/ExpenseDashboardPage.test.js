import React from 'react';
import ReactShallowRenderer from 'react-test-renderer/shallow';
import DashboardPage from '../../components/DashboardPage';

test('should display Dashboard with no items', () => {
    const renderer = new ReactShallowRenderer();
    renderer.render(<DashboardPage />)
    expect(renderer.getRenderOutput()).toMatchSnapshot();
})