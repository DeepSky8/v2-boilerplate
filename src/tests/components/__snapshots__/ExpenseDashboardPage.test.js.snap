// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should display Dashboard with no items 1`] = `
<div>
  Dashboard Page Content
</div>
`;
